src/frontend/src/apis/finance.api.ts

```diff
@@ -193,6 +183,36 @@ export const downloadBlobsToPdf = async (blobData: Blob[], filename: string) =>
       width,
       height
     });
+  };
+
+  // Embed the image in the PDF document
+  const promises = blobData.map(async (blob: Blob) => {
+    const arrayBuffer = await blob.arrayBuffer();
+    let image;
+    switch (blob.type) {
+      case 'image/jpeg': {
+        image = await pdfDoc.embedJpg(arrayBuffer);
```

Comment:
since ur never using it outside of these cases just initialize image here as a const